components/pages/explorer/RepoTable.tsx

```diff
@@ -225,6 +230,24 @@ const getTableStyle = (theme: DMSThemeInterface) => css`
   }
 `;
 
+const VariantsFileIdCell = (props: any) => {
+  const { NEXT_PUBLIC_BASE_PATH } = getConfig();
+  const value = get(props, "original.occurrence.hits.edges[0].node.file.object_id");
+  const fileLink = urlJoin(NEXT_PUBLIC_BASE_PATH, `${FILES_PATH}/?sqon={"op":"and","content":[{"op":"in","content":{"field":"object_id","value":["${value}"]}}]}`);
+  return value ? (
+    <StyledLink href={fileLink}>{value}</StyledLink>
+  ) : null;
+};
+
+const FilesObjectIdCell = (props: any) => {
```

Comment:
Just to help organize the code instead of making long files, can we move the two custom cells to their own files, put them in a folder for custom cells? Lets keep this file dedicated to the RepoTable component.